localstack/utils/bootstrap.py

```diff
@@ -2,6 +2,7 @@
 import re
 import sys
 import json
+import yaml
```

Comment:
Looks like CI builds are failing due to this import. We should either do a `pip install pyyaml` in the CI build, or simply move this import into the `validate_localstack_config(..)` function below (probably the latter is easier...)
https://travis-ci.org/github/localstack/localstack/builds/760733765